changelogs/fragments/195-add-interface-wireless-data.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - api_find_and_modify, api_info, api_modify - add support for ``interface wireless`` (https://github.com/ansible-collections/community.routeros/pull/195).
```

```suggestion
  - api_info, api_modify - add support for the ``interface wireless`` path (https://github.com/ansible-collections/community.routeros/pull/195).
```
`api_find_and_modify` does not use plugins/module_utils/_api_data.py.

---

Ah, right. It only imports `split_path`. I misread the code. Fixed.